services/ux-backend/handler/onboarding.go

```diff
@@ -0,0 +1,103 @@
+package handler
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/red-hat-storage/ocs-operator/v4/services/types"
+)
+
+func OnboardingTokenHandler(w http.ResponseWriter, r *http.Request) {
+	switch r.Method {
+	case "POST":
+
+		response, err := getOnboardingToken()
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+
+		w.Header().Set("Content-Type", "application/json")
+		//nolint
+		w.Write(response)
+
+	default:
+		fmt.Fprintf(w, "Sorry, only POST methods are supported.")
+	}
+}
+
+func getOnboardingToken() ([]byte, error) {
+
+	expirationHour := os.Getenv("ExpirationHour")
+	if expirationHour == "" {
+		expirationHour = "48"
+	}
+
+	expireHour, err := strconv.Atoi(expirationHour)
+	if err != nil {
+		return nil, err
+	}
+
+	data := types.OnboardingTicket{
+		ID:             uuid.New().String(),
+		ExpirationDate: time.Now().Add(time.Duration(expireHour) * time.Hour).Unix(),
+	}
+
+	payload, err := json.Marshal(data)
```

Comment:
Data and payload are synonyms, either find better names or just use the struct on location  
```suggestion
        expirationDate := time.Now().Add(time.Duration(expireHour) * time.Hour).Unix()
	payload, err := json.Marshal(types.OnboardingTicket{
		ID:             uuid.New().String(),
		ExpirationDate: expirationDate,
	})
```